ChangeLog

```diff
@@ -1,3 +1,26 @@
+2017-09-13  James Joseph Balamuta  <balamut2@illinois.edu>
+
+	* inst/rmarkdown/templates/pdf/skeleton/skeleton.Rmd: Quick pass over
+	content.
+	
+	* R/pinp.R: Added references and see also section.
+	* man/pinp.Rd: idem
+	
+	* inst/rmarkdown/templates/pdf/resources/pinp.bib: Added citations
+	for rticles, rmarkdown, and knitr. Changed PeerJ key away from DOI.
+	* inst/rmarkdown/templates/pdf/skeleton/skeleton.Rmd: Updated and 
+	added citations in skeleton
+	
+	* inst/rmarkdown/templates/pdf/resources/template.tex: Moved in 
+	bibliography creation from the line at end of file.
+	* inst/rmarkdown/templates/pdf/skeleton/skeleton.Rmd: Removed 
+	single line bibliography entry
+
+	* inst/rmarkdown/templates/pdf/skeleton/skeleton.Rmd: Bumped default
+	height of figure based on Dirk's suggestion.
+	
+	* man/pinp.Rd: Re-ran roxygen2 to pick up citation_package parameter added.
+	
```

Comment:
Thanks for adding these.  More fine-grained than mine (ie I skip trivial / tiny issues and aim for bigger picture), but we'll get hang.

Once 0.0.1 is up I also add a NEWS.Rd as usual.